### PR TITLE
chore: Leverage isEmpty vs size equality zero checks

### PR DESCRIPTION
**Pull request by kingthorin**
- Use isEmpty() instead, as pointed out by SAST (SonarLint)

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Will add comments to the 5 paros files :wink:

**Comment by thc202** [comment]
Please remove also `size() != 0`.

**Comment by kingthorin** [comment]
Done

File: zap/src/main/java/org/parosproxy/paros/core/proxy/ProxyParam.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
 // ZAP: 2019/06/01 Normalise line endings.
 // ZAP: 2019/06/05 Normalise format/style.
 // ZAP: 2020/11/26 Use Log4j 2 classes for logging.
+// ZAP: 2022/02/08: Use isEmpty where applicable.
```
Comment (thc202) [inline zap/src/main/java/org/parosproxy/paros/core/proxy/ProxyParam.java:45]: Remove the colon?

**Comment by kingthorin** [comment]
Tweaked

**Review by thc202 — APPROVED** [review]

**Review by psiinon — APPROVED** [review]

**Pull request merged**